src/commands/delete.ts

```diff
@@ -34,18 +31,21 @@ export async function deleteCommand(projectId: string, options: any) {
     }
 }
 
-async function deleteProjectHandler(projectId: string, forced: boolean) {
+async function deleteProjectHandler(projectId: string, forced: boolean): Promise<boolean> {
     // show prompt if no project id is selected
     if (typeof projectId === "string" && projectId.trim().length === 0) {
         const spinner = new Spinner("%s  ");
         spinner.setSpinnerString("|/-\\");
         spinner.start();
-        const projectsJson = await listProjects();
+        const projectsJson = await listProjects().catch((error) => {
+            throw error;
+        });
+
```

Comment:
This is uncessary. Any awaited async functions throw automatically if there is no `try catch` block around them, or have `.catch` call attached to them.

You can test it yourself here:
https://www.typescriptlang.org/play?#code/IYZwngdgxgBAZgV2gFwJYHsI2QCwE7oDuIAongXgBQCUMA3gFAzPb5EwBEApuenhwG4GAXwYNQkWIhQYsAE3QBldAFsuuVBADmNekxbBCwVMlYFiZCjRFiJ0eEihpMMFcYi7GLbHjB7v3obGpgrKahraNELewjBQwMhQODCUPHi0XgHMUJgg6AA2XAB0+eg6adTRLKKiDG6a1kA

```suggestion
        const projectsJson = await listProjects();
```

Reply:
I didn't know that - thanks for the playground. I reverted the catch blocks.